.prettierrc.js

```diff
@@ -0,0 +1,16 @@
+// Import the default config file and expose it in the project root.
+// Useful for editor integrations.
+const wpPrettierConfig = require( '@wordpress/prettier-config' );
+
+module.exports = {
+	...wpPrettierConfig,
+	overrides: [
+		{
+			files: '*.yml',
+			options: {
+				useTabs: false,
+				tabWidth: 2,
+			},
+		},
+	],
+};
```

Comment:
Why do we need this override?

Reply:
@swissspidy When both `.editorconfig` and `.prettierrc` are present, Prettier gives precedence to` .prettierrc`. Further the recommended indentation for YAML files is two spaces per level, so we override it here.

Reply:
But the default Prettier config should be correct for our needs, and the editorconfig should match the Prettier config 🤔

Reply:
@swissspidy The default Prettier configuration should suffice for our needs, and ensuring that the `.editorconfig `aligns with it eliminates the need for overrides. I have remove this overrides and now the .yml files formating looks similar to one in gutenberg project [ref](https://github.com/WordPress/gutenberg/blob/trunk/.github/dependabot.yml).